tests/templates/kuttl/smoke/requirements.txt

```diff
@@ -0,0 +1 @@
+trino==0.315.0
```

Comment:
Trino has its own dependencies, which aren't specified here. You can generate the proper list by running the following script:

```bash
python -m venv scrap
. scrap/bin/activate
pip install -r requirements.in
pip freeze > requirements.txt
deactivate
rm -r scrap
```

Reply:
https://pip-tools.readthedocs.io/en/latest/ can also automate this workflow for you.

Reply:
pinned dependencies